LondonBritishLibrary/orient/BLorient767.xml

```diff
@@ -30,20 +31,49 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 263</idno>
-                            
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
-                        <summary/>
-                        
+<msItem xml:id="ms_i1">
+<locus from="1r"/>
+<title xml:lang="gez" type="complete" ref="LIT2169Pateri">History of the fathers</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i2">
+<locus from="127r"/>
+<title xml:lang="gez" type="complete" ref="LIT6457DersanMacarius">A discourse on the life of Macarius by ʾabba Yāʿǝqob</title>
```

Comment:
Yāʿqob (or I would just leave out the written title and stick with the ID, but both are fine :) )

Reply:
I myself feel that having the written title is somewhat more user-friendly